### ISSUE TITLE
Close all issues

### ISSUE THREAD

**Issue by marcinofulus** [OpenDreamKit/OpenDreamKit#298]

**Comment by tthsqe12** [comment]
Before you close all issues, I think it would be great to have a faq section on the odk website where I can ask questions on how to set up a vre.

**Comment by alex-konovalov** [comment]
I think we may close them eventually, but no rush with that and no bulk closure - please do not flood people's notifications stream. One should close them gradually one by one and see if there are any follow-up questions coming from comments.

**Comment by wbhart** [comment]
The reason they are closing all the issues is that the ODK final review took place in Luxembourg yesterday. We haven't announced the final result yet as far as I know. A press release had been planned.....

Give people time [1] to recover from hangovers/sleep deprivation (especially our tireless leader @nthiery for the latter).

With a couple of followup proposals, a number of people still unemployed and no "pockets full of millions of Euros" ....

Even so, as I understand it, the website itself will be up until at least 2025 to keep the dream alive and to serve the community. Look out for an announcement and further details as soon as people recover. We know information about the final outcome and future of ODK is highly anticipated in the community. I'm sure those in charge of that will do their best to respond to the inevitable flood of requests as soon as possible.

[1] I myself who have been waking up at 7:15 am, collapsed into bed last night after getting off the train, didn't wake up until 11:30 today. Some other participants will probably need a couple of months to recover from sleep deprivation.

**Comment by nthiery** [comment]
Yeah, the adrenaline is settling down. I still woke up at 6am this morning, but am starting to feel the sleepiness :-)

I reworded the title; the idea is to close the issues for deliverables that were accepted (all but one :-) ), the corresponding tasks, and things that are not relevant any more. Will do in the coming days.
